### ISSUE TITLE
After manual brightness change, it changes automatically regardless of related settings 

### ISSUE THREAD

**Issue by Juriyx** [xanderfrangos/twinkle-tray#555]
There are two options in the Twinkle Tray app for setting up the brightness. The first one is located in the "General" section  - "Apply brightness at startup". The second option is located in the "Time Adjustments" section - "Check at app startup".  Also, I set up one time in the morning to change the brightness on to monitors to desired levels.

The current behavior: when I manually change the brightness using the Twinkle Tray icon in the system tray, it then always changes according to the time specified, regardless of whether both options are disabled or enabled.

The desired behavior: when the first option is enabled and the second is disabled, brightness should change only at the time specified. If I manually change the brightness, it should keep this level until the next scheduled time.

**Comment by TheConfax** [comment]
Did you solve? I have the same problem and just noticed this after opening my Issue #564

**Comment by ohycr** [comment]
I have the same problem too.

**Comment by Juriyx** [comment]
> Did you solve? I have the same problem and just noticed this after opening my Issue #564 

No, had to disable this and use manual mouse and keyboard control

**Comment by ohycr** [comment]
It can severely impact the experience.

**Comment by WesselKroos** [comment]
I have the same issue.

@xanderfrangos Is it correct that Twinkle tray updates the monitor brightness at a regular interval or certain system events, instead of only at the times set in the Time of Day Adjustments panel? If so, I think this problem could be easily fixed by providing a checkbox/switch in the system tray panel, so that we can temporarly toggle the Time of Day schedule off sometimes?

Something like this? (This toggle should only be visible to users that have a schedule)
![image](https://github.com/xanderfrangos/twinkle-tray/assets/31220528/f9aa0864-5a62-4854-b404-5c91b5677339)

**Comment by xanderfrangos** [comment]
Hey all,

This is by design. The automatic adjustments kick in again because I don't really have a way of knowing when a user may want to resume the automatic adjustments. Should it be different for manually adjusting it through the flyout versus hotkeys? Should regular behavior resume after 5 minutes? 15? The next adjustment time? What if the next time isn't for 10 hours and they want it to resume sooner automatically because they frequently make minor adjustments for specific apps? 

@WesselKroos
There's a toggle in the right-click menu to temporarily disable time adjustments:

<img width="300" alt="Screenshot 2023-07-22 135136" src="https://github.com/xanderfrangos/twinkle-tray/assets/33106561/22f3cb0c-c600-473f-8fbc-1bfd72533083">

(Disregard the DEBUG option)

Users have asked me to add all sorts of features, including time adjustments. They don't all work the way individual users want. It's difficult to determine what and how to add features that users ask for without cluttering the UI with excessive options. The Settings window is already overstuffed, and I'm trying not to make it worse. Your suggestion to show the toggle in the flyout isn't a bad one, but I need to weigh it with other features I'm working on. In the meantime, the toggle exists by right-clicking on the tray icon.

**Comment by WesselKroos** [comment]
@xanderfrangos I didn't know this toggle was already available from a right click. Thanks for the quick response!

I guess this issue can be closed then if this is by design.

**Comment by TheConfax** [comment]
@xanderfrangos can you take a look at my issue #564? I don't get if it's the same thing or a bug... mine seems a bug to me.

**Comment by ohycr** [comment]
Often turn into strange numbers
